src/Components/StatusBar/StatusBarMenu/StatusBarMenu.js

```
@@ -0,0 +1,125 @@
+                        width: `${MENU_WIDTH}px`,
+                    }}
+                >
+
+                    {threads &&
+                        threads.map((value, index) =>
+                            <option
+                                className="px-2"
+                                key={index}
+                                onClick={() => selectThread(index)}
+                            >Thread ID: {value}</option>
+                        )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)